provider/pkg/provider/local/command.go

```diff
@@ -58,8 +58,12 @@ func (c *CommandInputs) Annotate(a infer.Annotator) {
 	c.BaseInputs.Annotate(a)
 	a.Describe(&c.Triggers, "Trigger replacements on changes to this input.")
 	a.Describe(&c.Create, "The command to run on create.")
-	a.Describe(&c.Delete, "The command to run on delete.")
-	a.Describe(&c.Update, "The command to run on update, if empty, create will run again.")
+	a.Describe(&c.Delete, "The command to run on delete. The environment variables PULUMI_COMMAND_STDOUT "+
+		"and PULUMI_COMMAND_STDERR are set to the stdout and stderr properties of the Command resource from "+
+		"previous create or update steps.")
+	a.Describe(&c.Update, "The command to run on update, if empty, create will run again. The environment "+
+		"variables PULUMI_COMMAND_STDOUT and PULUMI_COMMAND_STDERR are set to the stdout and stderr properties "+
+		"of the Command resource from previous create or update steps.")
```

Comment:
I would insert some newlines in the description. Our codegen doesn't break up long lines in comments, so it's helpful to have them here. 

```suggestion
	a.Describe(&c.Delete, `The command to run on delete. The environment variables PULUMI_COMMAND_STDOUT
	and PULUMI_COMMAND_STDERR are set to the stdout and stderr properties of the
	Command resource from previous create or update steps.`)
	a.Describe(&c.Update, `The command to run on update, if empty, create will 
run again. The environment  variables PULUMI_COMMAND_STDOUT and PULUMI_COMMAND_STDERR 
are set to the stdout and stderr properties of the Command resource from previous 
create or update steps.`)
```